ghost/i18n/locales/lv/comments.json

```diff
@@ -0,0 +1,71 @@
+{
+    "{{amount}} characters left": "Atlikušas {{amount}} rakstzīmes",
+    "{{amount}} comments": "{{amount}} komentāri",
+    "{{amount}} hrs ago": "pirms {{amount}} stundam",
+    "{{amount}} mins ago": "Pirms {{amount}} minūtēm",
+    "{{amount}} more": "Vēl {{amount}}",
+    "1 comment": "1 komentārs",
+    "Add comment": "Pievienot komentāru",
+    "Add context to your comment, share your name and expertise to foster a healthy discussion.": "Pievienojiet savam komentāram kontekstu, kopīgojiet savu vārdu un zināšanas, lai veicinātu veselīgu diskusiju.",
+    "Add reply": "Pievienot atbildi",
+    "Already a member?": "Vai jau esat dalībnieks?",
+    "Anonymous": "Anonīms",
+    "Are you sure?": "Vai esat pārliecināts?",
+    "Become a member of {{publication}} to start commenting.": "Kļūstiet par {{publication}} dalībnieku, lai sāktu komentēt.",
+    "Become a paid member of {{publication}} to start commenting.": "Kļūstiet par apmaksātu {{publication}} dalībnieku, lai sāktu komentēt.",
+    "Cancel": "Atcelt",
+    "Comment": "komentēt",
+    "Complete your profile": "Pabeidziet savu profilu",
+    "Delete": "Dzēst",
+    "Deleted": "Izdzēsts",
+    "Deleted member": "Dzēsts dalībnieks",
+    "Deleting": "Dzēšana",
+    "Discussion": "Diskusija",
+    "Edit": "Rediģēt",
+    "Edit this comment": "Rediģēt šo komentāru",
+    "edited": "rediģēts",
+    "Enter your name": "Ievadiet savu vārdu",
+    "Expertise": "Ekspertīze",
+    "Founder @ Acme Inc": "Dibinātājs @ Acme Inc",
+    "Full-time parent": "Pilnas slodzes vecāks",
+    "Head of Marketing at Acme, Inc": "Mārketinga vadītājs uzņēmumā Acme, Inc",
+    "Hidden for members": "Paslēpts dalībniekiem",
+    "Hide": "Slēpt",
+    "Hide comment": "Slēpt komentāru",
+    "Jamie Larson": "Džeimijs Larsons",

```

Comment:
This is meant to be a 'generic' or common name in the language.  Rather than transliterating, could you please select a name that would be a good generic placeholder and would help show users that they should enter their full name there?